contrib/initramfs/scripts/zfs.in

```diff
@@ -895,9 +895,13 @@ mountroot()
 	    }' |
 	    while read i
 	do
-		if grep -sq noop /sys/block/$i/queue/scheduler
+		SCHEDULER=/sys/block/$i/queue/scheduler
+		if [ -e "$SCHEDULER" ]
 		then
-			echo noop > "/sys/block/$i/queue/scheduler"
+			case $(cat "$SCHEDULER") in
```

Comment:
Should there be quotes around `$(cat "$SCHEDULER")` here? It works fine and shellcheck isn't complaining, but that was surprising to me. It's probably fine, though?